tutorials/data-science-exploration/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Exploratory queries with Google BigQuery
+description: Use BigQuery to investigate arbitrarily large datasets.
+author: jerjou
+tags: Data Science, BigQuery
+date_published: 2017-05-10
+---
+[Google BigQuery](/bigquery) is a fully-managed service for data
+exploration using an SQL syntax. Once data has been loaded into the service, it
+can be queried at interactive speeds, without having to know beforehand what
+you're looking for.
+
+There has been much already written about using BigQuery for data analysis - for
+example:
+
+* [Analyzing Financial Time Series using BigQuery](/solutions/time-series/bigquery-financial-forex)
+* [How to forecast demand with Google BigQuery, public datasets and TensorFlow](/blog/big-data/2016/05/how-to-forecast-demand-with-google-bigquery-public-datasets-and-tensorflow)
+* Any of the articles around the [BigQuery Public Datasets](/bigquery/public-data/)
+
+For the purposes of this tutorial, we'll focus on finding meaning in the data
+that we've ingested from [previous](preprocessing.md) [articles](extraction.md).
```

Comment:
These two links right together without clarification could be confusing. Perhaps you could be more explicit about the articles while still maintaining the flow of the sentence, for example:
In this tutorial, you'll explore the data that you [preprocessed](preprocessing.md) and [extracted](extraction.md) as part of other tutorials in this series.